index.html

```diff
@@ -214,18 +214,18 @@ <h1 class="heading">Hey There. I'm Jamila</h1>
           </div>
         </div>
       </div>
-    </div>
+    </section>
 
     <!--SECTION ABOUT ME-->
-    <div class="section-about-me">
+    <div section="section-about-me">
       <div class="aboutme-block">
-        <div class="about-me-heading">About me</div>
-        <div class="about-me-desc">
+        <h2 class="about-me-heading">About me</h2>
+        <p class="about-me-desc">
           Hello I’m a software developer! I can help you build a product ,
           feature or website Look through some of my work and experience! If you
           like what you see and have a project you need coded, don’t hestiate to
           contact me.
-        </div>
+        </p>
         <div class="resume-btn">
           <a href="#" id="resume">Get My Resume</a>
         </div>
```

Comment:
- [ ] As suggested by the previous reviewer. The buttons should have some hovering styling changes. I noticed you implemented it on the technologies. However, you need to add the same hovering effect to the buttons including (project cards button, the get resume button).
Please work on this change.

See demo test:|
:------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/159134614-7bff7262-fb7d-4197-9372-439c54575cc3.gif)

Reply:
Added